src/lib.rs

```diff
@@ -67,9 +94,26 @@ mod tests {
         DateOfBirth(u32, u32, u32),
     }
 
+    #[allow(dead_code)]
+    #[derive(EnumDisplay)]
+    enum TestEnumWithGenerics<'a, T: Clone>
```

Comment:
Should this struct be called `TestEnumWithLifetimeAndGenerics`? Enforcing that each generic have a lifetime doesn't make much sense.

Reply:
We should probably have tests for both.